policyengine_us/parameters/gov/states/az/tax/income/deductions/standard/amount.yaml

```diff
@@ -10,18 +10,25 @@ metadata:
     href: https://azdor.gov/forms/individual/form-140-arizona-resident-personal-income-tax-booklet
   - title: 2021 Arizona Form 140 Resident Personal Income Tax Booklet
     href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2021_140BOOKLET.pdf#page=43
+  - title: 2023 Arizona Form 140 Resident Personal Income Tax Booklet
+    href: https://azdor.gov/forms/individual/form-140a-arizona-resident-personal-income-tax-booklet
+
 JOINT:
   2021-01-01: 25_100
   2022-01-01: 25_900
+  2023-01-01: 27_700
 HEAD_OF_HOUSEHOLD:
   2021-01-01: 18_800
   2022-01-01: 19_400
+  2023-01-01: 20_800
 WIDOW:
   2021-01-01: 25_100
   2022-01-01: 25_900
```

Comment:
Did they change the widow filing rules? In this tax form they should be head of household, not joint (page 9).

<img width="358" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/6076111/333248bb-0b98-4dee-a8b5-507bd08e2cde">

We should make an `az_filing_status` variable that incorporates this logic.